### PR TITLE
Menu page

### PR DESCRIPTION
**Pull request by BrassoiCatalin**
We can navigate through multiple possible pages; buttons in master page have styles.

### CODE REVIEW

File: SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                <StackLayout Margin="5, 30, 5, 5">
+                    <Button Text="Menu Item 1" 
+                            BackgroundColor="MediumVioletRed" 
+                            Clicked="Button_Clicked"
+                            CornerRadius="20"/>
+                    <Button Text="Menu Item 2" 
+                            BackgroundColor="MediumVioletRed"
+                            Clicked="Button_Clicked_1"
+                            CornerRadius="20"/>
+
```
Comment (TinelValentin) [inline SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml:27]: Isn't there some sort of general style so we do not have duplicated code?

File: SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            </ContentPage.Content>
+        </ContentPage>
+    </MasterDetailPage.Master>
+
+    <MasterDetailPage.Detail>
+        <ContentPage Padding="10">
+            <ContentPage.Content>
+                <StackLayout Margin="5, 30, 5, 5">
+                    <Label Text="Detail Page">
+
```
Comment (TinelValentin) [inline SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml:38]: Be more careful with those new lines!! ヽ༼ ಠ益ಠ ༽ﾉ

File: SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml (hunk 1 above)
Comment (BrassoiCatalin) [reply SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml:27]: Haven't searched that yet, but will do in a future possible task.

File: SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml (hunk 2 above)
Comment (BrassoiCatalin) [reply SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml:38]: Omg...

File: SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml (hunk 1 above)
Comment (BrassoiCatalin) [reply SevenSlots/SevenSlots/SevenSlots/View/MainPage.xaml:27]: Done! Buttons have now a common global style!

**Review by TinelValentin — APPROVED** [review]
Good Job!

**Review by Sorin37 — APPROVED** [review]

**Pull request merged**